modules/dcache-nfs/src/main/java/org/dcache/chimera/nfsv41/door/NFSv41Door.java

```diff
@@ -4,6 +4,7 @@
 import com.google.common.collect.Lists;
 import com.google.common.collect.Sets;
 import com.google.common.util.concurrent.ListenableFuture;
+import org.dcache.util.*;
```

Comment:
Sorry, one last thing: our coding-style doesn't allow for stars in non-static imports.

Having said that, the imports are already non-conforming since they are not sorted *sigh*.

I suggest you update the patch so that it doesn't convert the imports into a star-import.  Of course the patch still need to add the Glob import.  Just put that where the majority of org.dcache.util imports are location; i.e., after `import org.dcache.util.FireAndForgetTask;`